client/src/js/todos/addCard.js

```diff
@@ -0,0 +1,72 @@
+import { $, closest, containClass } from '../util';
+
+export class AddCard {
+  constructor(newCard, postedCard) {
+    this.target = $('main');
+    {
+      this.newCardTemplate = newCard;
+      this.postCardTemplate = postedCard;
+    }
+  }
+
+  init() {
+    this.target.addEventListener('click', this.addCardEventHandler);
+    this.target.addEventListener('input', this.inputFocusHandler);
+  }
+
+  addCardHandler = (e) => {
+    const columnList = $('.column-list', closest('.column', e.target));
+    $('.active', columnList)
+      ? columnList.firstElementChild.remove()
+      : columnList.insertAdjacentHTML('afterbegin', this.newCardTemplate());
+  };
+
+  cancelAddHandler = (e) => {
+    const columnList = $('.column-list', closest('.column', e.target));
+    columnList.firstElementChild.remove();
+  };
+
+  postCardHandler = (target) => {
+    const content = $('.item-content', closest('.list_item', target));
+    const maxLength = 200;
+    content.value.length > maxLength
+      ? alert('글자수를 초과하였습니다.')
+      : this.renderPostedCard(target);
+  };
+
+  inputFocusHandler = (e) => {
+    const accentBtn = $('.accent-btn', closest('.list_item', e.target));
+    const thisItem = closest('.list_item', e.target);
+    const title = $('.item-title', thisItem).value;
+    const content = $('.item-content', thisItem).value;
+    title && content
+      ? accentBtn.removeAttribute('disabled')
+      : (accentBtn.disabled = 'true');
+  };
+
+  renderPostedCard = (target) => {
+    const thisItem = closest('.list_item', target);
```

Comment:
네이밍 저도 너무 어려운데요. this, the 같은 접두사는 의미가 포괄적이어서 인지가 어려운 경우가 생길수도 있을 것 같습니다~ 의미를 조금더 담아보면 좋을 것 같아요!

```suggestion
    const cardItem = closest('.list_item', target);
```